File: web-app/src/main/java/pro/friendlyted/allmusic/generator/Writer.java
package pro.friendlyted.allmusic.generator;

import javax.sound.midi.MidiSystem;
import javax.sound.midi.Sequence;
import java.io.IOException;
import java.io.OutputStream;

public class Writer {

    public void save(OutputStream out, Sequence sequence) throws IOException {
        MidiSystem.write(sequence, 1, out);
    }
}

File: music-generator/src/main/kotlin/pro.friendlyted.ears.music.generator/Writer.kt
package pro.friendlyted.ears.music.generator

import javax.sound.midi.MidiSystem
import javax.sound.midi.Sequence
import java.io.IOException
import java.io.OutputStream

class Writer {

    @Throws(IOException::class)
    fun save(out: OutputStream, sequence: Sequence) {
        MidiSystem.write(sequence, 1, out)
    }
}
